physics/discrete_trajectory_body.hpp

```diff
@@ -108,19 +108,41 @@ template<typename Frame>
 void DiscreteTrajectory<Frame>::AttachFork(
     not_null<std::unique_ptr<DiscreteTrajectory<Frame>>> fork) {
   CHECK(fork->is_root());
-  CHECK(!fork->timeline_.empty());
+  CHECK(!Empty());
```

Comment:
This breaks the flight plan.
https://github.com/mockingbirdnest/Principia/blob/d682ca43aa569ef475b0a8e96c7c5315d4b9da81/ksp_plugin/flight_plan.cpp#L115-L121